yarn-project/circuits.js/src/constants.gen.ts

```diff
@@ -165,6 +165,7 @@ export const NUM_BASE_PARITY_PER_ROOT_PARITY = 4;
 export const RECURSIVE_PROOF_LENGTH = 93;
 export const NESTED_RECURSIVE_PROOF_LENGTH = 109;
 export const VERIFICATION_KEY_LENGTH_IN_FIELDS = 114;
+export const TUBE_VERIFICATION_KEY_LENGTH_IN_FIELDS = 103;
```

Comment:
The tube VK is basically Honk VK, this is not relevant for the purpose of generating a tube proof but we need to count this when we link the tube with honk verification